packages/blade/src/components/Table/TablePagination.web.tsx

```diff
@@ -242,7 +243,7 @@ const _TablePagination = ({
       display="flex"
       flexDirection="row"
       padding={tablePagination.padding}
-      backgroundColor={tablePagination.backgroundColor}
```

Comment:
Small change request: Can we also remove `backgroundColor` key from `tablePagination` object if its not being used anymore?

Reply:
how I can miss this 🤦. Thanks @saurabhdaware  🙌, done.